File: Practice/Daily_Practice/2024-12-05/Question_1.c
#include <stdio.h>
int simpleFunction(int Y[], int n, int x) {
    int total = Y[0], loopindex;
    for (loopindex = 1; loopindex <= n - 1; loopindex++) {
        total = x * total + Y[loopindex];
    }
    return total;
}

int main() {
    int Z[10] = {1, 1, 1, 1, 1, 1, 1, 1, 1, 1};
    int result = simpleFunction(Z, 10, 2);
    printf("%d", result);
    return 0;
}

/*
    O/P
    1023
*/

File: Practice/Daily_Practice/2024-12-11/Question_1.java
class Question_1 {
    public static float f(long m, long n) {
        float result = (float)m / (float)n;
        if (m < 0 | n < 0) return 0.0f;
        else result -= f(m * 2, n * 3);
        return result;
    }

    public static void main(String args[]) {
        System.out.println(f(1, 3));
    }
}

/*
    O/P
    2.0
*/